### PR TITLE
Home screen

### PR DESCRIPTION
**Pull request by JowelTisso**
* This PR is for the home screen
* Components -
    * Side nav
        * User info
        * Nav list
        * New post button
    * Right nav
        * Suggestion to follow
        * Notifications
    * Main content
        * Post component
        * User posts feed

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]
clean and modular code 
Good file structure

**Review by shraddha-amber — APPROVED** [review]
Very clean and beautiful UI
Good code structure

**Review by sanjitsarkar — COMMENTED** [review]
- Code is well-written and readable.
- Simple and clean UI
 Good to merge

**Review by jeetsdev — APPROVED** [review]
- Clean and modular code 
- Good folder structure 
- Descriptive class names

File: src/components/rightnav/Rightnav.jsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+      avatar: "https://i.pravatar.cc/150?img=31",
+    },
+  ];
+
+  return (
+    <aside className="rightnav flex-center pd-2x">
+      <p className="t4 section-title"> Suggestions</p>
+      <section className="nav-item right-nav">
+        {users.map(({ firstName, lastName, username, avatar }) => (
+          <main className="item-user pd-2x">
```
Comment (poornima-alugubelly) [inline src/components/rightnav/Rightnav.jsx:43]: oh damn I tried to do destructuring in map many times it never worked , so u r supposed to put () around the destructure , nice! learned something now

File: src/pages/home/Home.jsx [hunk 2]
```diff
@@ -1,22 +1,48 @@
...
 const Home = () => {
+  const [posts, setPosts] = useState([]);
+  const home = useSelector((state) => state.home);
   const dispatch = useDispatch();
+
+  useEffect(() => {
+    (async () => {
+      try {
+        const res = await GET(API.ALL_POST);
+        if (res?.status === 200 || res?.status === 201) {
```
Comment (poornima-alugubelly) [inline src/pages/home/Home.jsx:20]: liked how u made the API url also a constant

File: src/utils/Constant.js [hunk 3]
```diff
@@ -5,6 +5,7 @@ export const USER_TOKEN = "USER_TOKEN";
 export const API = Object.freeze({
```
Comment (poornima-alugubelly) [inline src/utils/Constant.js:5]: learnt about object.freeze from ur code , nice!

**Review by poornima-alugubelly — APPROVED** [review]

**Review by simran61 — APPROVED** [review]
clean and modular code
Good code structure

**Pull request merged**